anchor/anchorRecord.go

```diff
@@ -207,11 +239,20 @@ func UnmarshalAndValidateAnchorEntryAnyVersion(entry interfaces.IEBEntry, public
 	return ar, valid, err
 }
 
-func CreateAnchorRecordFromDBlock(dBlock interfaces.IDirectoryBlock) *AnchorRecord {
-	ar := new(AnchorRecord)
-	ar.AnchorRecordVer = 1
-	ar.DBHeight = dBlock.GetHeader().GetDBHeight()
-	ar.KeyMR = dBlock.DatabasePrimaryIndex().String()
-	ar.RecordHeight = ar.DBHeight
-	return ar
+// IsSame returns true iff all fields of BitcoinStructs bc==bc2
```

Comment:
typo `iff`

Reply:
iff = if and only if , it was already in use in the file, so I continued its usage, but happy to clean up to be more explicit if necessary

Reply:
iff is fine :+1: